### ISSUE TITLE
JsValue::into_serde() is deprecated in wasm-bindgen 0.2.83

### ISSUE THREAD

**Issue by cjpatton** [cloudflare/daphne#118]
I got the following warning when publishing daphne_worker 0.1.2 (I can't seem to reproduce when building locally):

```
warning: use of deprecated associated function `worker::wasm_bindgen::JsValue::into_serde`: causes dependency cycles, use `serde-wasm-bindgen` or `gloo_utils::format::JsValueSerdeExt` instead
  --> src/durable/garbage_collector.rs:85:38
   |
85 |                         item.value().into_serde()?;
   |                                      ^^^^^^^^^^
   |
   = note: `#[warn(deprecated)]` on by default

warning: use of deprecated associated function `worker::wasm_bindgen::JsValue::into_serde`: causes dependency cycles, use `serde-wasm-bindgen` or `gloo_utils::format::JsValueSerdeExt` instead
  --> src/durable/leader_agg_job_queue.rs:88:38
   |
88 |                         item.value().into_serde()?;
   |                                      ^^^^^^^^^^

warning: use of deprecated associated function `worker::wasm_bindgen::JsValue::into_serde`: causes dependency cycles, use `serde-wasm-bindgen` or `gloo_utils::format::JsValueSerdeExt` instead
   --> src/durable/leader_col_job_queue.rs:125:38
    |
125 |                         item.value().into_serde()?;
    |                                      ^^^^^^^^^^

warning: use of deprecated associated function `worker::wasm_bindgen::JsValue::into_serde`: causes dependency cycles, use `serde-wasm-bindgen` or `gloo_utils::format::JsValueSerdeExt` instead
   --> src/durable/report_store.rs:117:76
    |
117 |                     let (key, report_hex): (String, String) = item.value().into_serde()?;
    |                                                                            ^^^^^^^^^^

warning: `daphne_worker` (lib) generated 4 warnings
    Finished dev [unoptimized + debuginfo] target(s) in 49.95s
   Uploading daphne_worker v0.1.2 (/Users/chris/github.com/cloudflare/daphne/daphne_worker)
warning: aborting upload due to dry run
```

Happily, serde-wasm-bindgen is maintained by Cloudflare, so it may be a matter of integrating into workers-rs: https://github.com/cloudflare/serde-wasm-bindgen

**Comment by cjpatton** [comment]
wasm-bindgen is re-exported by worker; we don't actually use it directly. Consider upgrading it there. https://github.com/cloudflare/workers-rs/issues/237.

**Comment by bhalleycf** [comment]
Closing as this was fixed by #237 .
